newscoop/classes/SaaS.php

```diff
@@ -42,8 +42,8 @@ public static function singleton()
      */
     public function hasPermission($p_permissionString)
     {
-        if (array_key_exists('permissions', $this->saasConfig)) {
-            $permissions = $this->saasConfig['permissions'];
+        if (array_key_exists('permissions', self::$saasConfig)) {
+            $permissions = self::$saasConfig['permissions'];
```

Comment:
Check https://github.com/ahilles107/Newscoop/blob/e76c2548d225fc72181f8bc607b40e22ae33b33a/newscoop/classes/SaaS.php#L76 it is still not changed here.